### DISCUSSION TITLE
1st run of german translation of missing strings. [DE]

### DISCUSSION THREAD

**Discussion by ansgarKf** [aclap-dev/video-downloadhelper#152] [Language & Translation]
how about these translation of missing strings ? However: try to translate weh_prefs_label_tbvwsExtractionMethod==Extraction method might mislead. Further infos about this item possible ? Or digging in source code needed ? re-run possible.
(it might be: "not the yellow of the egg")
[VDH++messages.json](https://github.com/aclap-dev/video-downloadhelper/files/14319645/VDH%2B%2Bmessages.json)

**Comment by Harry123456789012** [comment]
Translation to German:
weh_prefs_label_tbvwsExtractionMethod==Herauslöse-Methode könnte irreführend sein. Weitere Informationen über diesen Punkt möglich? Oder Quellcode einsehen erforderlich? Neustart möglich.

**Reply by Erdal70** [reply]
I think, [Harry123456789012] translated it really good. I want to use "Extraktionsmethode" or "Methode des Extrahierens" instead of "Herauslöse-Methode":
weh_prefs_label_tbvwsExtractionMethod==Extraktionsmethode könnte irreführend sein. Weitere Informationen über diesen Punkt möglich? Oder Quellcode einsehen erforderlich? Neustart möglich.

**Comment by ansgarKf** [comment]
EN: "Extraktionsmethode" is a 'word-by-word' "Translation" of "ExtractionMethod". On my point of view even "Entnahme-Methode" interprets this label quite unspecific. (what is really meant?)  Further info about this might help. Translations arent cut in stone.
DE: "Extraktionsmethode" ist eine wort-wörtliche 'Übersetzung' von "ExtractionMethod". Mit geringen Hintergrund-Infos über das angegebene label ist das Ergebnis eher unspezifisch/ungenau. Weitere Angaben wären hilfreich.

**Comment by paulrouget** [comment]
Can you guys come to an agreement on what needs to happen here?

**Comment by ansgarKf** [comment]
FR: Bon soir Paul, utilizer "Extraktions-Methode" maintenant, svp. 
EN: Have a good day Paul, pls use "Extraktions-Methode" now.
DE: Guten Tag Paul, verwende bitte jetzt "Extraktions-Methode".
EN: When the day will come, the origin about "weh_prefs_label_tbvws" will reveal. Which describes the label better.
Or will be better fitting.
